yatsm/cli/pixel.py

```diff
@@ -190,15 +191,27 @@ def pixel(ctx, config, px, py, band, plot, ylim, style, cmap,
         )
 
 
-def plot_TS(dates, y):
+def plot_TS(dates, y, seasons):
     """ Create a standard timeseries plot
 
     Args:
         dates (iterable): sequence of datetime
         y (np.ndarray): variable to plot
     """
     # Plot data
-    plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    if seasons == []:
+        plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    else:
+        for date in dates:
+            if date.month == 11 or date.month == 12 or date.month == 1 or date.month == 2 or date.month == 3:
```

Comment:
You probably were tripped up by the tuple definition in Python -- for a single value you need a trialing comma. I forget this all the time...

``` python
a = (5)  # int
a = (5, )  # tuple of 1 int
```